assets/src/components/detourListPage.tsx

```diff
@@ -93,7 +27,14 @@ export const DetourListPage = () => {
           Add detour
         </Button>
       )}
-      <DetoursTable data={fakeData} />
+      {detours && (
+        <>
+          {detours.active && <DetoursTable data={detours.active} />}
+          {detours.draft && <DetoursTable data={detours.draft} />}
+          {detours.past && <DetoursTable data={detours.past} />}
```

Comment:
```suggestion
          <h2>Active detours</h2>
          {detours.active && <DetoursTable data={detours.active} />}
          <h2>Draft detours</h2>
          {detours.draft && <DetoursTable data={detours.draft} />}
          <h2>Closed detours</h2>
          {detours.past && <DetoursTable data={detours.past} />}
```

Non-blocking, but I had to add this in on my local machine in order to be able to understand what I was seeing, and I figured it might be helpful.

No big deal though - something like this will happen sooner or later as we move closer and closer to the hi-fi designs.

Reply:
Right, I figured this will be done in your hi-fi work. Fine to add it here, too

Reply:
Nevermind, rolling back because it wasn't worth the resulting CI issues and was out-of-scope